<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>492</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>492</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/rsz_logo.png);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="mainStack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>471</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background:transparent;
      border: 2px solid gray;
      border-radius: 10px;
color: rgb(23, 109, 2);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="userName">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>202</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="placeholderText">
        <string>Give Your User Name</string>
       </property>
      </widget>
      <widget class="QPushButton" name="registerButton_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>310</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:blue;
color:white;</string>
       </property>
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="firstName">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>202</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Give Your First Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lastName">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>50</y>
         <width>202</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="placeholderText">
        <string>Give Your Last Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="mobileNum">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>202</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Give Your Mobile No.</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="eMail">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>202</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="placeholderText">
        <string>Give Your E-mail ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="getPass">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>202</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Give Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="conPass">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>250</y>
         <width>202</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Re-Type your Password</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>431</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:blue;</string>
      </property>
      <property name="text">
       <string>NO USER FOUND! PLEASE REGISTER TO CONTINUE...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>364</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/exit.jpg);
border-radius: 20px;
color:white;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>471</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> background:transparent;
color: rgb(23, 109, 2);
      border: 2px solid gray;
      border-radius: 10px;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="login_username">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="login_password">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginButton_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>230</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid #8f8f91;
border-radius: 6px;
background:blue;
color:white;
</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
